src/main/java/org/sonatype/nexus/repository/composer/internal/ComposerRecipeSupport.groovy

```diff
@@ -63,6 +75,15 @@ abstract class ComposerRecipeSupport
   @Inject
   Provider<AttributesFacet> attributesFacet
 
+  @Inject
+  Provider<SingleAssetComponentMaintenance> componentMaintenanceFacet
+
+  @Inject
+  Provider<HttpClientFacet> httpClientFacet
```

Comment:
Question: Is this used anywhere?

Reply:
> Question: Is this used anywhere?

It's used in the proxy recipe and was added to the hosted recipe, but we don't really need it in the hosted recipe for now. I lifted it up to the recipe support class because it was used in both places, but instead I'm just taking it out and pushing it back down to the proxy recipe for now:

https://github.com/sonatype-nexus-community/nexus-repository-composer/pull/12/commits/c1d24fd0f725a807031d463004897bcbc0aced28